sparrowpy/form_factor/integration.py

```diff
@@ -467,6 +459,23 @@ def _area_under_curve(ps: np.ndarray, order=2) -> float:
 
     return area
 
+def _booles_rule(x,y):
```

Comment:
looks like pure numpy, maybe we can accolorate it and the method which is calling it with numba.